CMake/cdat_modules/vtk_external.cmake

```diff
@@ -26,21 +26,21 @@ list(APPEND vtk_build_args
 )
 
 # Use cdat zlib
-if(NOT CDAT_USE_SYSTEM_ZLIB)
-  list(APPEND vtk_build_args
-    -DZLIB_INCLUDE_DIR:PATH=${cdat_EXTERNALS}/include
-    -DZLIB_LIBRARY:FILEPATH=${cdat_EXTERNALS}/lib/libz${_LINK_LIBRARY_SUFFIX}
-  )
-endif()
+#if(NOT CDAT_USE_SYSTEM_ZLIB)
```

Comment:
yes I turned off zlib and libxml2 libxslt etc.. there are on every system. I guess we can uncomment if you want but I'm not positive about what happen to CDAT_USE_SYSTEM_ZLIB now that we don't have include(libz_pkg)